quantum/stm32/chconf.h

```diff
@@ -58,7 +58,7 @@
  *          The value one is not valid, timeouts are rounded up to
  *          this value.
  */
-#define CH_CFG_ST_TIMEDELTA                 2
+#define CH_CFG_ST_TIMEDELTA                 0
```

Comment:
This may (does not) need to be changed if `STM32_PWM_USE_TIM2` is set to false;

https://github.com/qmk/qmk_firmware/blob/b2283d76c18f79783bd14f536ff89e82df852594/quantum/stm32/mcuconf.h#L186

Or if `STM32_ST_USE_TIMER` was changed to something other than 2.

Reply:
This change was also a result of my suggestion to @jackhumbert back in the day. Reverting this change causes firmware to be uncompilable, so a bunch of other things needs to be changed so that firmware can be compiled again. Unfortunately playing with timers i couldn't find the right combination for all of this to work without this change. I wrote the same thing on my PR, but i guess yours is gonna be merged instead of mine (or at least before mine) that's why i'm writing it here as well.

Reply:
this one is explicitly required to be set to true for every timer used with pwm.